test/new-e2e/tests/npm/dockercompose_httpbin.go

```diff
@@ -0,0 +1,31 @@
+package npm
+
+import (
+	"github.com/DataDog/test-infra-definitions/components/docker"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"gopkg.in/yaml.v2"
+)
+
+func dockerHTTPBinCompose() docker.ComposeInlineManifest {
+	httpbinManifestContent := pulumi.All().ApplyT(func(args []interface{}) (string, error) {
+		agentManifest := docker.ComposeManifest{
+			Version: "3.9",
+			Services: map[string]docker.ComposeManifestService{
+				"httpbin": {
+					Privileged:    true,
+					Image:         "mccutchen/go-httpbin",
+					ContainerName: "httpbin",
+					Pid:           "host",
+					Ports:         []string{"80:8080/tcp"},
+				},
+			},
+		}
+		data, err := yaml.Marshal(agentManifest)
+		return string(data), err
+	}).(pulumi.StringOutput)
+
+	return docker.ComposeInlineManifest{
+		Name:    "httpbin",
+		Content: httpbinManifestContent,
+	}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Should be possible to load a manifest using a file, embedding it with `embed`, and pass it to the docker manager

Reply:
You can load it as a string and use the `pulumi.String` to convert it to the expectdd `Content` type

Reply:
@pducolin seems we need a top level mapping object
got the error `
            error: Process exited with status 15: running " docker-compose -f $HOME/httpbin-compose-tmp/docker-compose-httpbin.yml up --detach --wait --timeout 300":
            Top-level object must be a mapping`

Reply:
seems related to the double quote issue https://stackoverflow.com/questions/46550348/docker-stack-deploy-error-about-top-level-object-mappings